roles/base/templates/run-ansible-pull.sh.j2

```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
```

Comment:
Perhaps but it works in FreeBSD's sh.

    $ /bin/sh
    $ RETRY=0
    $ RETRY=$((RETRY+1))
    $ echo $RETRY
    1